hs/src/Reach/JSUtil.hs

```diff
@@ -173,6 +174,15 @@ mkCommaTrailingList xs = JSCTLComma (toJSCL xs) JSNoAnnot
 a2sp :: JSAnnot -> JSSemi
 a2sp = JSSemi
 
+getFormals :: JSExpression -> [JSExpression]
+getFormals = \case
```

Comment:
There's gotta be some other code that does this... so you should make it use this function